hive-event-listeners/apiary-gluesync-listener/pom.xml

```diff
@@ -108,6 +135,7 @@
         <version>3.1.1</version>
         <executions>
           <execution>
+            <id>shade-all</id>
```

Comment:
why do we need shade-all + shade-full also what's really the difference between all and full?

Reply:
I'm not sure
@javsanbel2 do we need this?

Reply:
The easiest example is, I need all the metastore classes.

Reply:
I had thought of renaming it as 'cli' instead of 'full', would that make it clearer?

Reply:
Yeah let's call it cli

Reply:
renamed to cli.